WordPress/Classes/ViewRelated/Stats/SiteStatsDashboardViewController.swift

```diff
@@ -364,7 +364,12 @@ struct SiteStatsDashboardPreferences {
         guard let siteID = SiteStatsInformation.sharedInstance.siteID?.intValue else { return nil }
 
         let key = Self.lastSelectedStatsTabTypeKey(forSiteID: siteID)
-        return StatsTabType(rawValue: UserPersistentStoreFactory.instance().integer(forKey: key))
+
+        guard let tabRawValue = UserPersistentStoreFactory.instance().value(forKey: key) as? Int else {
```

Comment:
nitpick:

```suggestion
        guard let tabRawValue = UserPersistentStoreFactory.instance().object(forKey: key) as? Int else {
```

While this works, I think the `value(forKey:)` function is from `NSObject` [here](https://developer.apple.com/documentation/objectivec/nsobject/1412591-value), so it would be more correct to use `UserDefaults`'s [`object(forKey:)`](https://developer.apple.com/documentation/foundation/userdefaults/1410095-object) instead.

With this change, the app compiles without the method added to the `UserPersistentRepositoryReader` protocol. Maybe the compiler checks yje classes that conform to `UserPersistentRepository` and found they both have this method:
- `UserDefaults` ✅ (built-in)
- `InMemoryUserDefaults` ✅